generator/autogenlist.ts

```diff
@@ -498,6 +498,10 @@ const autogenlist: AutogenlistConfig[] = [
         basePath: 'synapse/resource-manager',
         namespace: 'Microsoft.Synapse',
     },
+    {
+        basePath: 'security/resource-manager',
+        namespace: 'Microsoft.Security',
+    },
```

Comment:
For all the resource types marked as "Unknown Scope", you'll need to add config here to annotate which scopes they are valid at. This is because we weren't able to unambiguously detect the scope from the URLs in your swagger specs.

From your output, that looks like:
```
- advancedThreatProtectionSettings
- deviceSecurityGroups
- informationProtectionPolicies
- assessments
- deviceSecurityGroups
- iotSensors
```

Reply:
pushed a change based on this